security/s2a/internal/crypter/aesgcm_test.go

```diff
@@ -21,8 +21,9 @@ package crypter
 import (
 	"bytes"
 	"fmt"
-	"google.golang.org/grpc/security/s2a/internal/crypter/testutil"
 	"testing"
+
+	"google.golang.org/grpc/security/s2a/internal/crypter/testutil"
```

Comment:
Why did these get rearranged?

Reply:
interestingly, it seems that emacs automatically rearranges them when I save. I'll try to find a way to prevent that.

Reply:
Strange, it could be that we're using different go linter versions.

Reply:
As long as it was just the linter, that's fine.